core/test/equations_test.jl

```diff
@@ -31,6 +31,7 @@
     # The storage of the basin when it has the same level as the level boundary
     limit_storage = 450.0
     decay_rate = -1 / (basin_area * p.linear_resistance.resistance[1])
+    # TODO adapt for max_flow_rate of 6e-5
     storage_analytic =
         @. limit_storage + (storage[1] - limit_storage) * exp.(decay_rate * t)
```

Comment:
@SouthEndMusic it would be great if you could help updating this test value.

Reply:
Solve for Q(level) == max_flow_rate

Find out at which times it switches from max_flow_rate to dynamic flow rate

Compute from there